pom.xml

```
@@ -9,7 +9,7 @@
 	</parent>
 	<groupId>life.qbic</groupId>
 	<artifactId>projectbrowser-portlet</artifactId>
-	<version>1.11.4</version>
+	<version>1.12.0-SNAPSHOT</version>
```

This should be 1.12.0
```suggestion
	<version>1.12.0</version>
```